### DISCUSSION TITLE
Saved in the cloud, where?

### DISCUSSION THREAD

**Discussion by sawa-ko** [ebenbruyns/junkstore#51] [Q&A]
I've seen the latest post about saving in the cloud, but it doesn't specify where. Do you refer to cloud storage of the Epic Games Store itself or of its own servers?

Sorry if the question is somewhat silly, only that in the post it does not explicitly specify where.

**Comment by ebenbruyns** [comment]
The cloud storage is whatever epic provides, this only uses the legendary functionality. The post also comes with a big warning that we don't trust the code, so please be aware of this as you'll need the skill to troubleshoot this yourself as we're explicitly not supporting this for the time being. This will provide similar functionality to heroic, but caveat emptor.

**Reply by sawa-ko** [reply]
Thanks for reply.

Oh, I understand, I guess since it's saved in the cloud, does the achievement system also work or is that a separate issue?

**Reply by mrsjunkrunner** [reply]
Sorry about not replying to you last year! To answer your question about the achivement system for Epic, yes it will still work. Make sure you have installed the EOS overlay from the slider icon in the Epic tab. Then you will need to enable it for games that require it - like Fall Guys and Rocket League. 

But if you are only playing games on your own and don't need the social aspect of it then it will still track your achievments - for games like Freshly Frosted, Vampire Survivors, CYGNI, etc.
